### PR TITLE
Terminal fixes: use bash, UI improvements

### PR DESCRIPTION
**Pull request by yebrahim**
This fix adds a bashrc file inside the container to set up the enviornment for the user. Currently it changes directory to `/content`, colors the prompt, and adds a message to tell the user to keep sensitive work in `/content`. The `ENV` env var is set so that file gets loaded by all shells.

The rest of the PR is UI fixes to make things look better, and fixed the order of loading the Jupyter dialog module.

image

### CODE REVIEW

File: sources/web/datalab/templates/terminals.html [hunk 1]
```diff
@@ -60,9 +59,8 @@
         1
         2
         3
-        4
-      <span id="dummy-screen-rows" style="">01234567890123456789012345678901234567890123456789012345678901234567890123456789</span>
-      </pre>
+        </pre>
+      <span id="dummy-screen-rows" style="">01234567890123456789012345678901234567890123456789012345678901234567890123456789012</span>
```
Comment (jimmc) [inline sources/web/datalab/templates/terminals.html]: Why did you change from 80 to 83 characters here?

Comment (yebrahim) [inline sources/web/datalab/templates/terminals.html]: 80 characters wasn't well aligned with the size of the terminal. Unfortunately we have little control over this calculation because of the so many parameters it uses:
- Window dimensions
- Terminal container dimensions
- Line height
- Padding and margin

So I had to adjust it until it fit well with our usage.

File: sources/web/datalab/templates/terminals.html [hunk 2]
```diff
@@ -29,13 +29,12 @@
     </div>
     <div id="header"></div>
     <div id="site">
-      <div id="terminado-container">
-      </div>
+      <div id="terminado-container"></div>
     </div>
     <div style="position:absolute; left:-1000em">
       <!-- The following elements contain dummy content that is used to calculate
       the size of the terminal characters. For more context, look at terminal/js/main.js -->
```
Comment (jimmc) [inline sources/web/datalab/templates/terminals.html]: Maybe "... that is used by hack code in Jupyter ..."

Comment (yebrahim) [inline sources/web/datalab/templates/terminals.html]: Sure.

Comment (jimmc) [inline sources/web/datalab/templates/terminals.html]: Was there a change here?

Comment (yebrahim) [inline sources/web/datalab/templates/terminals.html]: Oh, it seems I missed pushing the comment fix somehow!
